src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types/Person';
+
+interface Props {
+  person: Person | null;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
```

Comment:
```suggestion
export const PersonLink: React.FC<Props> = memo({ person }) => {
```
probably, I'm not sure